src/_sass/header2.scss

```diff
@@ -0,0 +1,204 @@
+$header2-width:    1080px;
+
+$color-white: #ffffff;
```

Comment:
can you move this into the existing color scale?

Reply:
I had it in the shared tilt.scss file, but I moved them into this header2.scss because there's actually a subtly different color scale which I'll be implementing along with the refresh of the entire Landing Page. I wanted to make it clear that this is not a part of the original layout (aka, what's in tilt.scss). 

Is it OK if I leave these inside header2 for now, and then further pull these out in a shared CSS which is clearly part of the new layout in the forthcoming Landing Page PR?

Reply:
ok. if we're going to do a new color scale, can we namespace them in some way? e.g., $color2-...

I'm pretty sure that moving them into header2.scss doesn't solve the problem? There's no file-level scope in SASS. So it doesn't really matter what file they're in. Setting a global variable just overwrites the other one for future declarations, which makes this harder to reason about

Reply:
Good point! I know that there's no file-level scope in SASS, but I hoped that keeping them in header2.scss would show they weren't for general use. That said, it wasn't explicit enough, so I added a prefix ("header2") and also added a comment.